pkg/ast/pipesearch/searchHandler.go

```diff
@@ -451,8 +452,28 @@ func convertRRCsToJSONResponse(rrcs []*sutils.RecordResultContainer, sizeLimit u
 	if sizeLimit < uint64(len(allJsons)) {
 		allJsons = allJsons[:sizeLimit]
 	}
+	highlightSearchTerms(qid, qc.SearchTerms, allJsons)
 	return allJsons, allCols, nil
 }
+func highlightSearchTerms(qid uint64, searchTerms []string, allJsons []map[string]interface{}) {
```

Comment:
It looks like this function will highlight all occurrences of the searchTerms; is that what you want? I thought for something like `foo=bar` we'd only want to highlight `bar` if it's in the `foo` column